components/table-accordion-group/src/test.js

```diff
@@ -16,7 +16,7 @@ describe('TableAccordionGroup', () => {
   });
 
   it('renders style rules', () => {
-    expect(wrapper).toHaveStyleRule('font-family', 'Roboto,Arial,sans-serif');
+    expect(wrapper).toHaveStyleRule('font-family', '"nta",Arial,sans-serif');
```

Comment:
this should probably use the constant from @govuk-react/constants

Reply:
It could do, but it wouldn't be checking much if it did? I can definitely update it tho.

Reply:
my thought was really that as this is checking that the font-family matches the constant, this test would be fragile if we change the constant in govuk-react

not super-important tho :grin: